rules/react/walmart.js

```diff
@@ -11,12 +11,22 @@ module.exports = {
   "rules": {
     // Prevent missing displayName in a React component definition
     "react/display-name": 0,
+    // Forbid certain propTypes
+    "react/forbid-prop-types": 0,
     // Enforce boolean attributes notation in JSX
     "react/jsx-boolean-value": 2,
+    // Validate closing bracket location in JSX
+    "react/jsx-closing-bracket-location": 0,
     // Enforce or disallow spaces inside of curly braces in JSX attributes
     "react/jsx-curly-spacing": 0,
+    // Validate props indentation in JSX
+    "react/jsx-indent-props": 0,
+    // Limit maximum of props on a single line in JSX
+    "react/jsx-max-props-per-line": 0,
```

Comment:
We're already gated by line length here. I think this is a good thing to think about but maybe not something we should programmatically enforce.